app/domain_models/record_time_create.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'common_error'
+
+class RecordTimeCreate
+
+    attr_accessor :user_id, :contents, :duration, :distance, :user
+    
+    def initialize(user_id, contents, duration, distance)
+        @user_id             = user_id
+        @contents            = contents
+        @duration            = duration
+        @distance            = distance
+        @user                = User.find_by(id: user_id)
+    end 
+
+    def create_time_record #新增time-based的健身紀錄
+        no_user_found!
```

Comment:
no_user_found
You do not have to do it here.
It is in LoginUser.

Reply:
Deleted